lib/chromium.js

```diff
@@ -28,6 +29,42 @@ function broutHandler() {
   });
 }
 
+function checkPortAvailability(port, cb) {
```

Comment:
Instead of adding code that checks for available ports to this package, I would recommend using an existing package.

[`portscanner`](https://github.com/baalexander/node-portscanner) looks promising.

Reply:
Looks like this doesn't even need a dependency as `.listen()` will automatically assign a free port when you pass 0: https://nodejs.org/dist/latest-v8.x/docs/api/net.html#net_server_listen